swagger/swagger.yaml

```diff
@@ -147,5 +147,31 @@ paths:
           description: "Successful data fetch of rackmount catalogs"
       x-tags:
       - tag: "default"
-
+  /serviceProfile:
+    get:
+      tags:
+      - "default"
+      description: "Returns an array of Service Profiles"
+      operationId: "controllers.default_controller.getServiceProfile"
+      parameters:
+      - name: "host"
+        in: "query"
+        description: "USC manager host name or IP"
+        required: false
```

Comment:
From the UcsHandle implementation (https://github.com/CiscoUcs/ucsmsdk/blob/master/ucsmsdk/ucshandle.py), I don't think the host/user/password can be optional, so why you define them as optional?
If this is a bug, please fix it, If this is purposely, please explain the reason.